src/pricing/cc-pricing-header.js

```diff
@@ -167,25 +168,37 @@ export class CcPricingHeader extends LitElement {
 
         sl-select {
           --focus-ring: 0 0 0 .2em rgba(50, 115, 220, .25);
-          --sl-input-background-color-disabled: #eee;
+          --sl-input-background-color: var(--cc-color-bg-default, #fff);
+          --sl-input-background-color-disabled: var(--cc-color-bg-neutral-disabled, #eee);
+          --sl-input-background-color-hover: var(--cc-color-bg-default, #fff);
+          --sl-input-background-color-focus: var(--cc-color-bg-default, #fff);
```

Comment:
this is to prepare for dark mode mainly. We didn't need to set these as long as our bg is white because it matches shoelace values but when we switch to dark, we need to override the shoelace defaults.